src/commonmark.rs

```diff
@@ -440,6 +440,68 @@ impl<'a, W: Write + 'a> Context<'a, W> {
         );
         write!(self.output.writer, "{}", osc(&command))
     }
+
+    /// Write an inline image for Terminology.
+    fn write_terminology_inline_image(&mut self, path: &Path) -> Result<()> {
+        fn get_lines(columns: u16, file: &str) -> u16 {
+            use std::process::Command;
+            use termion::terminal_size;
+
+            let lines = terminal_size().unwrap_or((0, 24)).1;
```

Comment:
Please move this to the `terminal` module, similar to the existing helpers for the column with.

Also, please don't compute it inline, but add it to the `Context`. There's already a field for the `columns`, why not replace it with a `size` field with a corresponding `struct Size`?